src/ColumnEncodingUtility/analyze-schema-compression.py

```diff
@@ -729,6 +731,12 @@ def analyze(table_info):
                 
                 # get the primary key statement
                 statements.extend([get_primary_key(analyze_schema, target_schema, table_name, target_table)]);
+
+                # set the table owner
+                statements.extend(['alter table %s."%s" owner to %s;' % (target_schema, target_table, owner)]);

```

Comment:
@IanMeyers I'm ok with qualifying this with a check if `--target-schema` is specified, but I'm not quite sure how to do it.  Python is not my native language.

Reply:
No problem - will fix it as I have another round of updates going in